code/__DEFINES/subsystems.dm

```diff
@@ -165,6 +166,7 @@
 #define FIRE_PRIORITY_ATMOS_ADJACENCY	300
 #define FIRE_PRIORITY_CHAT			400
 #define FIRE_PRIORITY_OVERLAYS		500
+#define FIRE_PRIORITY_EXPLOSIONS	666
```

Comment:
these arent tabbed like the other ones and it makes me angry

Reply:
Tabs for indentation, spaces for alignment.

Reply:
github doesnt always display tabwidths correctly